cpp/Platform.Data/WriteHandlerState.h

```diff
@@ -0,0 +1,36 @@
+namespace Platform::Data::Doublets
+{
+    using namespace Platform::Interfaces;
+    template<typename TStorage>
+    struct WriteHandlerState
+    {
+        typename TStorage::LinkAddressType Break;
+        typename TStorage::LinkAddressType Result;
+        std::function<typename TStorage::LinkAddressType(typename TStorage::HandlerParameterType, typename TStorage::HandlerParameterType)> Handler;
```

Comment:
Performance!!!
Pls, use `std::optional`

Reply:
How you suggest to use `std::optional` here?